backend/app/processing/llm.py

```
@@ -0,0 +1,369 @@
+        }
+        
+        payload = {
+            "inputs": prompt,
+            "parameters": {
+                "max_length": 100,
+                "temperature": 0.95,
+                "top_p": 0.9
+            }
+        }
+        
+        response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)